src/kernel/arch/x86.zig

```diff
@@ -0,0 +1,51 @@
+// Zig version: 0.4.0
```

Comment:
Do we need this and the x86_mock file now? I'm guessing that it's superseded by arch/x86/arch.zig. LGTM otherwise.

Reply:
These files got accidentally added when i was re-basing 100 times lol, have removed.